crates/holochain_zome_types/src/cell.rs

```diff
@@ -34,7 +34,7 @@ pub const CLONE_ID_DELIMITER: &str = ".";
 ///
 /// Example: `profiles.0`
 #[derive(Clone, Debug, Eq, Hash, PartialEq, serde::Serialize, serde::Deserialize)]
-pub struct CloneId(pub RoleName);
+pub struct CloneId(pub String);
```

Comment:
Not a `RoleName`, it's a custom formatted string

Reply:
also found that confusing